hubmap/frontend/package.json

```diff
@@ -5,19 +5,24 @@
   "dependencies": {
     "@babel/runtime": "^7.4.5",
     "@google/model-viewer": "^0.3.1",
-    "@material-ui/core": "^4.0.1",
+    "@material-ui/core": "^4.4.2",
     "@material-ui/icons": "^4.0.1",
     "@material-ui/styles": "^4.0.1",
+    "autosuggest-highlight": "^3.1.1",
     "axios": "^0.19.0",
     "contentful": "^7.6.0",
     "cross-fetch": "^3.0.4",
     "d3": "^5.9.2",
     "d3-selection": "^1.4.0",
+    "flat": "^4.1.0",
     "material-table": "^1.39.0",
+    "material-ui-chip-input": "^1.0.0",
     "mui-datatables": "^2.4.0",
     "node-sass": "^4.12.0",
     "prop-types": "^15.7.2",
+    "pubsub-js": "^1.7.0",
```

Comment:
Interesting. redux is something I'd be glad to use for like a middleware such as for wrapping api interactions. Even for middleware, PubSub is a handy package to send out some status. Given that we don't have an api yet, we should revisit to define what is required for "middleware" and for component-to-component interactions. We are not using any apis /redux for ui-components since everything is mockup from json.

Generally, I think selected options can filter multiple components: experiments list/table, summary, search and other pages also have similar functionality: on home page: if a search for gene expression started, would filter the charts, human anatomy map, summary, etc. So we should keep this as it is now and revisit later when we consider middleware design for react app. We always can **later**, if required, write custom pub-sub with custom global store rather than redux/pubssub-jsother popular packages. we do not have any active issues for middleware design in react right now, isn't it?